media/css/mozorg/home/home-a.less

```diff
@@ -0,0 +1,1097 @@
+        width: 100%;
+    }
+
+    #slide-two {
+        display: block;
+        background-image: url('/media/img/home/new/intro/intro-learning-600.jpg');
+    }
+
+    @media @mq-tablet {
+        .slide {
+            display: block;
+        }
+
+        #slide-one {
+            background-image: url('/media/img/home/new/intro/intro-mitchell-1000.jpg');
+        }
+
+        #slide-two {
+            background-image: url('/media/img/home/new/intro/intro-learning-1000.jpg');
+        }
+
+        #slide-three {
+            background-image: url('/media/img/home/new/intro/intro-advocacy-1000.jpg');
+        }
+    }
+
+    @media @mq-desktop-wide {
+        #slide-one {
+            background-image: url('/media/img/home/new/intro/intro-mitchell-2000.jpg');
+        }
+
+        #slide-two {
+            background-image: url('/media/img/home/new/intro/intro-learning-2000.jpg');
+        }
+
+        #slide-three {
+            background-image: url('/media/img/home/new/intro/intro-advocacy-2000.jpg');
+        }
+    }
+}
+
+
+// Mozilla in action
+.section-action {
+    .invert-text;
+    background: #00b086;
+}
+
+#encryption-video {
+    width: 100%;
+    height: auto;
+}
+
+.moz-video-container {
+    margin-bottom: 0;
+
+    &:after {
+        content: none;
+    }
+}
+
+.js .moz-video-button:after {
+    background: url('/media/img/home/new/video-play.svg');
+    opacity: 0.8;
+}
+
+
+// Web Innovations
+.section-innovation {
+    .invert-text;
+    background: saturate(darken(#24caff, 10%), 15%);
+}
+
+#aframe-wrapper {
+    background: #24caff url('/media/img/home/new/a-frame/aframe.png') center center no-repeat;
+    background-size: 533px 300px;
+    height: 300px;
+    position: relative;
+    width: 100%;
+
+    @media only @mq-tablet {
+        background-size: 889px 500px;
+        height: 500px;
+    }
+
+    @media only @mq-phone-wide {
+        background-size: 676px 380px;
+        height: 380px;
+    }
+}
+
+#aframe-iframe-shield {
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    z-index: 2;
+}
+
+#aframe-iframe {
+    height: 100%;
+    left: 0;
+    opacity: 0;
+    position: absolute;
+    top: 0;
+    transition: opacity 0.25s ease-out;
+    width: 100%;
+    z-index: 1;
+
+    &.loaded {
+        opacity: 1;
+    }
+}
+
+
+// Firefox
+.section-firefox {
+    .invert-text;
+    overflow: hidden;
+
+    h2 {
+        .at2x('/media/img/firefox/common/firefox-logo.png', 80px, 83px);
+        background-position: top center;
+        background-repeat: no-repeat;
+        margin-top: 40px;
+        padding-top: 93px;
+        text-align: center;
+        width: 180px;
+
+        @media only @mq-tablet {
+            background-size: 120px 125px;
+            padding-top: 135px;
+            width: 250px;
+        }
+    }
+}
+
+.event-horizon {
+    background-color: #0150ab;
+    background: url('/media/img/firefox/new/redesign/stars.png') center 80px no-repeat,
```

Comment:
This might be fine to leave in for now, but the design on /new is changing possibly as soon as the next sprint. We may just need to remember to move/rename the files for this page.